kuma/landing/jinja2/landing/includes/contribution-form.html

```diff
@@ -0,0 +1,52 @@
+<div class="contribution-form contribution-banner">
+    <div class="row">
+        <img src="{{ static('img/hero-dino-blank.png') }}" class="backdrop-image" alt="">
+        <div class="column">
+            <div class="container">
+                <h2 class="highlight highlight-spanned">
+                    <span class="highlight-span">{{ _('Help Support MDN for as little as $2')}}</span>
+                </h2>
+                <h4>{{_('We need your help to grow to the next stage.')}}</h4>
+                <p class="body">{{_('Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud.')}}</p>
+                {% if is_cta %}
+                <div class="button-group form-footer">
+                    <a href="#" class="cta-link">
+                        {{_('Learn More')}} {% include 'includes/icons/arrows/arrow.svg' %}
+                    </a>
+                    <a href="#" class="cta-link">
+                        {{_('Feedback')}} {% include 'includes/icons/arrows/arrow.svg' %}
+                    </a>
+                </div>
+                {% endif %}
+            </div>
+        </div>
+        <div class="column">
+            <form id="contribute-form-2" class="container" method="post" action="">
+                {% csrf_token %}
+                <div class="form-group">
+                    {% for field in form %}
+                        {% if field.label and not field.hidden %}
+                            <span class="label"> {{field.label}} </span>
+                        {% endif %}
+                        {{ field }}
+                        {{ field.errors }}
+                    {% endfor %}
+                </div>
+                <div class="form-footer">
+                    <div class="payment-details">
+                        <img src="{{ static('img/mc-logo.png') }}" alt="mastercard">
```

Comment:
We're trying to use SVG instead of PNG when we can, but the SVG versions I found of these logos are bigger than the PNGs, so PNGs are OK.

Reply:
I think SVGs would make sense for higher density screens, I can look into image optimisation.